types/chromecast-caf-receiver/cast.framework.d.ts

```diff
@@ -1261,3 +1266,15 @@ export class AudioTracksManager {
     setActiveById(id: number): void;
     setActiveByLanguage(language: string): void;
 }
+
+/** Represents playback statistics */
+export class Stats {
```

Comment:
Can you copy and paste the documentation for this class and each property, please?

Reply:
They have a link to the docs in the description, or do you mean as a comment in the source itself?

Reply:
In the code itself. That will show it in the user's IDE autocompletion.